baselines/compute/src/v1/addresses_client.ts.baseline

```diff
@@ -453,7 +471,15 @@ export class AddressesClient {
       'project': request.project || '',
     });
     this.initialize();
-    return this.innerApiCalls.insert(request, options, callback);
+    return this.innerApiCalls.insert(request, options, callback)
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    .then(([response, next, rawResponse]: [any, any, any]) => {
+      return [
```

Comment:
This is exactly what I was picturing, I'm glad it wasn't too painful to make the data fit the appropriate shape.

Reply:
synced offline